bin/travis-build.bash

```diff
@@ -5,12 +5,14 @@ echo "This is travis-build.bash..."
 
 echo "Installing the packages that CKAN requires..."
 sudo apt-get update -qq
-sudo apt-get install postgresql-$PGVERSION solr-jetty libcommons-fileupload-java:amd64=1.2.2-1
+sudo apt-get install solr-jetty libcommons-fileupload-java
 
 echo "Installing CKAN and its Python dependencies..."
 git clone https://github.com/ckan/ckan
 cd ckan
-git checkout $CKAN_BRANCH
+export latest_ckan_release_branch=`git branch --all | grep remotes/origin/release-v | sort -r | sed 's/remotes\/origin\///g' | head -n 1`
+echo "CKAN branch: $latest_ckan_release_branch"
+git checkout $latest_ckan_release_branch
```

Comment:
I know that's what the extension template will output, but with this change we are losing the testing across different CKAN versions (`CKAN_VERSION` env var). For general purpose extensions like this one it's worth testing against different versions to ensure they work as expected. Here's an [example](https://github.com/frictionlessdata/ckanext-validation/blob/master/bin/travis-build.bash#L14:L19).

Reply:
That's something we ran into for ckanext-pages, too. See ckan/ckanext-pages#62.

Reply:
thanks for the hint @amercader and @torfsen !
I'll have a look into that probably end of this week.